assets/text/api/v1-compat/auth/cloud/basic-auth.js

```diff
@@ -0,0 +1,35 @@
+/**
+  * Use an InfluxDB Cloud username and token
+  * with Basic Authentication
+  * to query the InfluxDB 1.x-compatibility API
+  */
```

Comment:
When rendered, the actual code feels crowded by the comment. I'd just suggest adding new lines after the comment blocks in these code examples.

```suggestion
  */

```